features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/renderers/controllers/asset_router.js

```diff
@@ -48,10 +47,14 @@ var render=function(theme,data,meta,require){
             }
             data.newViewData.publishActionAuthorized = publishActionAuthorized;
             heading = data.newViewData.displayName.value;
-            data.businessOwnerName = businessOwnerName;
+            var businessOwnerAttribute = data.artifact.attributes.overview_businessOwner;
+            if (!businessOwnerAttribute.trim()) {
+                var businessOwner =  apiProvider.getBusinessOwner(businessOwnerAttribute);
+                data.businessOwnerViewData = require('/helpers/splitter.js').transform(businessOwner);
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5